src/client.ts

```diff
@@ -329,8 +329,20 @@ export class Client {
         return cb(`You may only request ${MAX_ADDR} addresses at once.`);
 
       const fwConstants = getFwVersionConst(this.fwVersion);
-      if (!isValidAssetPath(startPath, fwConstants))
+      const flags = fwConstants.getAddressFlags || [];
+      let isPubkeyOnly = false;
```

Comment:
We could set this boolean directly if we wanted to: 

```suggestion
      const isPubkeyOnly = flags.indexOf(flag) > -1 &&
        (
          flag === EXTERNAL.GET_ADDR_FLAGS.ED25519_PUB ||
          flag === EXTERNAL.GET_ADDR_FLAGS.SECP256K1_PUB
        )
```